scripts/queue-and-execute.ts

```diff
@@ -36,7 +36,7 @@ export async function queueAndExecute() {
     descriptionHash
   )
   await executeTx.wait(1)
-  console.log(await box.retrieve().toString())
+  console.log(`Box value: ${await box.retrieve()}`)
```

Comment:
```suggestion
  console.log(`Box value: ${await box.retrieve().toString()}`)
```

Reply:
It was printing out [Object object] with `toString()`.  Not sure if this is newer node runtime behaviour with object.prototype.toString() - but seems to date back to 2017 [see first reply in 1] and see below screenshot of output.

<img width="1433" alt="Screen Shot 2022-05-08 at 7 02 28 pm" src="https://user-images.githubusercontent.com/8016129/167289334-d2a6d20e-b8cf-4b30-b705-a2b92e9b1ffd.png">

By removing `toString()` I got it to provide the returned number.  JS magic?!

Happy to leave as is  -your call! 


[1] https://stackoverflow.com/questions/46146860/javascript-template-strings-dont-pretty-print-objects

Reply:
Yep, I'm being dumb. Thank you!